src/lava/magma/runtime/runtime.py

```diff
@@ -201,56 +205,97 @@ def start(self, run_condition: AbstractRunCondition):
         else:
             print("Runtime not initialized yet.")
 
-    def _run(self, run_condition):
+    def _get_resp_for_run(self):
+        """
+        Gets response from RuntimeServices
```

Comment:
What response for what? would you understand this if you saw this for the first time?
Remember, the audience for docstrings are not your pair-programming partner who is anyways intimately familiar with the code, but the random guy on the street.